packages/sku/src/services/packageManager/pnpmConfig.test.ts

```diff
@@ -1,10 +1,16 @@
-import { describe, it, vi } from 'vitest';
+import { describe, it, vi, beforeEach } from 'vitest';
 import { createFixture, type FileTree } from 'fs-fixture';
+import * as ci from '../../utils/isCI.js';
 
 import { validatePnpmConfig } from './pnpmConfig.js';
 
 describe('validatePnpmConfig', () => {
-  describe.for`
+  describe('when not in CI', () => {
+    beforeEach(() => {
+      vi.spyOn(ci, 'default', 'get').mockReturnValue(false);
+    });
```

Comment:
This is the recommended way of mocking exported values https://main.vitest.dev/guide/mocking.html#mock-exported-variables.